hub/core/dataset/dataset.py

```diff
@@ -2195,7 +2197,11 @@ def _save_view_in_user_queries_dataset(
 
         try:
             queries_ds = hub.load(
-                queries_ds_path, verbose=False
+                queries_ds_path,
+                verbose=False,
+                read_only=False,
```

Comment:
why the read_only is false by default?

Reply:
We have to raise an exception if write access to user's query folder is not available as soon as possible. If failing at a later stage, the traceback might not be clear.

Reply:
got it!